plugins/utils/ollama.nix

```diff
@@ -106,6 +106,7 @@ in
             Defaults to the value of `plugins.ollama.model`.
           '';
 
+          # FIXME should the default be { __raw = "```$ftype\n(.-)```"; }?
```

Comment:
Ok, let's do this then.

Reply:
Nope, I was wrong. It's a string not lua: https://github.com/nomnivore/ollama.nvim?tab=readme-ov-file#extracting